--- Edge-Application-Management.yaml	2024-09-19 16:25:36.291151681 -0700
+++ Edge-Application-Management.yaml.last	2024-09-19 16:25:31.470961850 -0700
@@ -732,7 +732,7 @@
           pattern: ^[A-Za-z][A-Za-z0-9_]{1,63}$
           description: Name of the application.
         version:
-          type: string
+          type: integer
           description: Application version information
         packageType:
           description: Format of the application image package
